### DISCUSSION TITLE
HTML的發明人：提摩西·約翰·柏內茲-李爵士 (Sir Timothy John Berners-Lee)

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#10] [General]
要有怎麼樣的機緣才可以創造出影響後世的HTML文件格式?

來看看 [HTML的發明人：提摩西·約翰·柏內茲-李爵士 (Sir Timothy John Berners-Lee) 的故事吧!](https://zh.wikipedia.org/wiki/%E8%92%82%E5%A7%86%C2%B7%E4%BC%AF%E7%BA%B3%E6%96%AF-%E6%9D%8E)

![image](https://user-images.githubusercontent.com/1768891/134809725-e1d6a4e6-ea38-4020-9bde-39a8e77d215e.png)

**Comment by boson13579** [comment]
讀完提姆·柏內茲-李的生平故事，我們可以發現他創造出[超文字系統]絕非偶然，先不提他的父母都是電腦科學家，在他擔任[歐洲核子研究組織]（CERN）擔任[獨立承包人]的那段時間裡，他便建立一個以[超文字系統]為基礎的專案，方便研究人員分享及更新訊息。
在1989年的時候，CERN是全歐洲最大的網際網路節點。柏內茲-李因此看到了將超文字系統與網際網路結合在一起的契機
超文字系統 就如此誕生了，提姆·柏內茲-李創造出[超文字系統]是偶然，也是必然，機會是留給準報好了人的!
